### PR TITLE
Disabled auto show

### PR DESCRIPTION
**Pull request by mantou132**
To avoid a white screen, need to disable the automatic show of windows.

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -115,7 +115,9 @@ impl<R: Runtime> WindowExt for Window<R> {
 }
 
 #[derive(Default)]
-pub struct Builder {}
+pub struct Builder {
+  disable_auto_show: bool,
```
Comment (amrbashir) [inline src/lib.rs]:
```suggestion
  auto_show: bool,
```

File: src/lib.rs [hunk 2]
```diff
@@ -67,11 +67,11 @@ impl<R: Runtime> AppHandleExt for tauri::AppHandle<R> {
 }
 
 pub trait WindowExt {
-  fn restore_state(&self) -> tauri::Result<()>;
+  fn restore_state(&self, disable_auto_show: bool) -> tauri::Result<()>;
```
Comment (amrbashir) [inline src/lib.rs]:
```suggestion
  fn restore_state(&self, auto_show: bool) -> tauri::Result<()>;
```

**Review by amrbashir — CHANGES_REQUESTED** [review]

File: src/lib.rs (hunk 1 above)
Comment (mantou132) [reply src/lib.rs]: Need to modify the default behavior?

Comment (amrbashir) [reply src/lib.rs]: just the naming of the fields

**Review by amrbashir — APPROVED** [review]

**Comment by amrbashir** [comment]
@mantou132 seems like you didn't sign your commits and I can't merge this until you rebase your PR with signed commits.

**Comment by mantou132** [comment]
> s like you didn't sign your commits and I can't merge this until you rebase your PR with signed commits.

Now I have added the signature

**Comment by amrbashir** [comment]
Thank you

**Pull request merged**